qutebrowser/browser/hints.py

```diff
@@ -65,24 +65,13 @@ class HintLabel(QLabel):
         _context: The current hinting context.
     """
 
-    STYLESHEET = """
-        QLabel {
-            background-color: {{ conf.colors.hints.bg }};
-            color: {{ conf.colors.hints.fg }};
-            font: {{ conf.fonts.hints }};
-            border: {{ conf.hints.border }};
-            padding-left: -3px;
-            padding-right: -3px;
-        }
-    """
-
     def __init__(self, elem, context):
         super().__init__(parent=context.tab)
         self._context = context
         self.elem = elem
 
         self.setAttribute(Qt.WA_StyledBackground, True)
-        config.set_register_stylesheet(self)
+        self.setObjectName("hint")
```

Comment:
Given that this is an object name, I think it's meant to be unique. Not sure what happens if we don't respect that - though we do the same in other places as well.

Reply:
I don't think that's an issue (even though it dosen't matter for this PR before). From the QObject docs for `findChild`:

> If there is more than one child matching the search, the most direct ancestor is returned. If there are several direct ancestors, it is undefined which one will be returned. In that case, findChildren() should be used.

Which implies that multiple objects can have the same name